.gitignore

```diff
@@ -5,6 +5,8 @@ dist
 lib
 es
 coverage
+# used by rollup-plugin-typescript2
+.rpt2_cache/
```

Comment:
I don't know why they do this by default. `node_modules/.cache` is the standard that a few other tools use now. 

Can we add `cacheRoot: path.resolve('./node_modules/.cache')` to the plugin options? Then we don't need ignore it specifically.

Reply:
nice. added and pushed

Reply:
Sent a PR upstream if anyone is interested: https://github.com/ezolenko/rollup-plugin-typescript2/pull/167